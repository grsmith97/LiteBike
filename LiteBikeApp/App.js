import React from 'react';
import {StyleSheet, View} from 'react-native';
import {StatusBar} from 'expo-status-bar';

import StartScreen from './src/screens/StartScreen';

export default function App() {
  return (
    <View style={styles.container}>
      <StatusBar style='auto' />
      <StartScreen />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
  },
});
