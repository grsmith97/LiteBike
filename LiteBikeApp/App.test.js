import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

let component;

describe('Given App component', () => {
  describe('When the component renders', () => {
    beforeEach(() => {
      component = renderer.create(<App />);
    });

    it('Then the component matches the snapshot', () => {
      expect(component.toJSON()).toMatchSnapshot();
    });

    it('Then Hello World appears on the screen', () => {
      const text = component.root.findByProps({'data-testing': 'hello-world'});
      expect(text.props.children).toBe('Hello, world!');
    });
  });
});
