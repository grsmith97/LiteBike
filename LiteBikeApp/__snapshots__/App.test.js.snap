// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given App component When the component renders Then the component matches the snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    data-testing="hello-world"
    style={
      Object {
        "fontSize": 30,
      }
    }
  >
    Hello, world!
  </Text>
</View>
`;
